packages/cli/package.json

```diff
@@ -14,8 +14,7 @@
     "test": "echo no tests yet -- # npm run test:ava",
     "test:ava": "ava",
     "preinstall": "tar --version 2>&1 >/dev/null",
-    "prepack": "node ../../scripts/publish.js",
-    "postbuild": "yarn prepack && cp ./src/completion ../../dist/cli/completion -rL && cp ./src/types/commander-completion.d.ts ../../dist/cli/types/commander-completion.d.ts -L"
```

Comment:
This will be brought back after it no longer causes CPM to break

Reply:
no prepack = no /dist/cli

Reply:
Yes, I will bring it back before merging. Actually I wanted to only get rid of postbuild for the time of iterating on the CPM PR